app/controllers/application_controller.rb

```diff
@@ -20,6 +20,10 @@ class ApplicationController < ActionController::Base
 
   before_action :set_timezone
 
+  # Allow users to impersonate other uses in a non-production environment.
+  # See the `pretender` gem.
+  impersonates :user
```

Comment:
We can certainly add more protection @carpodaster. I don't know if doing it here is the best idea though, because this call creates the `true_user` helpers (and a couple more).

For now, I think it would work because I think we're only using `true_user` after we check that we're not in production in the views. I'll try it out and see if it works!